### ISSUE TITLE
Libreoffice invisible after launch

### ISSUE THREAD

**Issue by ZechariahB** [micheleg/dash-to-dock#1784]
**Libreoffice** launches an application before launching **Base**, **Calc**, **Draw**, **Impress**, **Math**, or **Writer**. As such, only the first program launched is briefly visible before no application is displayed at all. Libreoffice is regardless visible in activities. After Libreoffice is launched, a workaround is to either reset the shell (which is impossible in Wayland), launch another application, or disable then enable the extension to update the dash. 

I tested the default ubuntu-dock extension provided in the 22.04 Jammy Jellyfish LTS release of Ubuntu as well as ubuntu-dock-jammy branch of this extension. This issue is very reproducible with any settings configured as far as I know. Please notify if other branches are affected as well.

**Comment by vanvugt** [comment]
This sounds like a general gnome-shell bug, possibly caused by other extensions. If you can reproduce it consistently then please run this command to report it to Ubuntu:
```
ubuntu-bug gnome-shell
```

**Comment by ZechariahB** [comment]
I don't believe this is a gnome shell bug yet, since I tested it with all extensions disabled prior to reporting this. There are no extensions that are affecting the dock abnormally. I enabled Window List for comparison, an extension part of Gnome Classic side by side with dash-to-dock and launched LibreOffice Writer. Window List displays LibreOffice Writer as intended. This behavior is likely specific to dash-to-dock and is always repeatable.
